src/BugsnagUnity/Configuration.cs

```diff
@@ -54,15 +55,53 @@ public class Configuration : IMetadataEditor, IFeatureFlagStore
 
         internal OrderedDictionary FeatureFlags = new OrderedDictionary();
 
+        private List<Regex> _redactedKeysRegex;
+        public string[] RedactedKeys = new string[] { "password" };
```

Comment:
The old matching was whole word and case insensitive - should we make this the same?